### PR TITLE
Add CONTRIBUTING.md, PR template, issue templates

### PR DESCRIPTION
**Pull request by nslee333**
Added a Contributing.md file, issue and PR templates. Also corrected an example command in the .env.example.

These changes will complete several tasks in #79

### CODE REVIEW

File: CONTRIBUTING.md
Comment (MattIPv4) [inline CONTRIBUTING.md:1]:
```suggestion
# Contributing to Alveus.gg
```
I think we can word this so it feels a bit more specific to the project

File: CONTRIBUTING.md
Comment (MattIPv4) [inline CONTRIBUTING.md:7]:
```suggestion
If you're looking to work on an issue but have a question about it or are generally unsure about how to approach it, reach out in that specific issue and folks will be able to help.

For general questions about the project unrelated to a specific issue, please start a [discussion](https://github.com/orgs/alveusgg/discussions) (or file an issue if you believe what you're asking is something actionable in code).
```
Rewording this to push folks towards issues primarily if there is an appropriate one, before falling back to discussions.

File: CONTRIBUTING.md
Comment (MattIPv4) [inline CONTRIBUTING.md:29]: Instead of having this written out here, I think we'd be better off setting up some [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) that prompt for these fields?

File: CONTRIBUTING.md
Comment (MattIPv4) [inline CONTRIBUTING.md:43]:
```suggestion
- Please keep your commit message to a concise message that clearly explains the change made.
```

File: CONTRIBUTING.md
Comment (MattIPv4) [inline CONTRIBUTING.md:51]: Similar to the issues note, we can create a [PR template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) (just Markdown, not a form) instead of having this in text.

Comment (nslee333) [reply CONTRIBUTING.md:51]: Cool, I'll look into these and see if I can get it setup.

**Comment by nslee333** [comment]
Alright I have the contributing.md suggestions added. Issue, and PR templates added as well.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (MattIPv4) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
## Describe your changes

<!-- If your changes resolve an open issue, please make sure to note that here using a GitHub keyword (https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) -- e.g. resolves #123 -->

...

## Notes for testing your change

...
```
I think we can just move the issue linking note into the change description, and we can make it a comment so that it doesn't show in the actual PR

File: CONTRIBUTING.md
Comment (MattIPv4) [inline CONTRIBUTING.md:5]:
```suggestion
## Questions
```
Nested heading levels

File: CONTRIBUTING.md
Comment (MattIPv4) [inline CONTRIBUTING.md:11]:
```suggestion
## Development
```
Ditto

**Comment by nslee333** [comment]
Alright I got the suggestions added - it's showing up as I committed it before your review, but I just added it. WSL2 has clock issues so that's why it showed up like that.

File: README.md
Comment (MattIPv4) [inline README.md:53]:
```suggestion
1. If you find a bug - you can fill out a bug [report](https://github.com/alveusgg/alveusgg/issues/new/choose)
```

File: README.md
Comment (MattIPv4) [inline README.md:54]:
```suggestion
2. If you have an idea that would make Alveus better - please fill out an idea [issue](https://github.com/alveusgg/alveusgg/issues/new/choose)
```

**Comment by MattIPv4** [comment]
I think other than that, this is probably good to go, unless you want to do more with it? Feel free to mark as ready for review whenever and we can give it a final once over before merging :)

**Comment by nslee333** [comment]
Alright cool, I don't think this is far off from knocking out #79 completely, I can add a Code of Conduct, just need a little input on what you guys want to do with that.

**Comment by MattIPv4** [comment]
If you're interested in figuring out what Code of Conduct makes sense, I think we'd probably want that as a PR into the .github repo in the org, so that it's shared across all the repos.

**Comment by nslee333** [comment]
Ah okay cool, sounds good.

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (pjeweb) [inline .github/ISSUE_TEMPLATE/bug_report.yml:6]: I am not sure if I understand this correctly, but i think this is not what we want? I think new reports should probably not be assigned at all by default.

File: .github/ISSUE_TEMPLATE/idea.yml
Comment (pjeweb) [inline .github/ISSUE_TEMPLATE/idea.yml:6]: dito

**Review by pjeweb — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (nslee333) [reply .github/ISSUE_TEMPLATE/bug_report.yml:6]: Definitely, I'll change it.

**Review by pjeweb — APPROVED** [review]
thank you for your work!

**Review by MattIPv4 — APPROVED** [review]

**Comment by MattIPv4** [comment]
Thank you for the work on this, it is very much appreciated! 💚

**Pull request merged**
